### ISSUE TITLE
vscode can`t respond when using mstsc.exe

### ISSUE THREAD

**Issue by Hukeqing** [microsoft/vscode#114067]
- VSCode Version: 1.52.1
- OS Version: Windows_NT x64 10.0.19042(double)

Steps to Reproduce:

1. Use mstsc(C:\WINDOWS\system32\mstsc.exe) on computer A connect to a computer B
2. Input CTRL+A on a file in vscode(computer B) by computer A
3. The vscode on computer B can`t respond this input but it looks like noting happened
4. Stop using mstsc.exe and open computer B
5. The vscode will not respond to any of my actions on computer B, but the system does not show that it has entered an unresponsive state. Sometimes this state will automatically return to normal after a period of time

Does this issue occur when all extensions are disabled?: Yes/No
No, but there is only one extension(Chinese (Simplified) Language Pack for vscode by Microsooft). Because I can`t disable it

**Comment by meganrogge** [comment]
Hi I'm not familIar with that program. What is it and how does it relate to vs code?

**Comment by Hukeqing** [comment]
Hello, mstsc.exe is a remote desktop application native to Windows system. It is located in C:\WINDOWS\system32\mstsc.exe. I usually use it to remote my computer at my work place. After I have used this native software, vs code will become unable to respond to my operations, and I need to restart vs code. Fortunately, I have not had this problem recently. I am sorry that I cannot give more details about this problem until I encounter this problem next time.

**Comment by Hukeqing** [comment]
I'm not sure if this is caused by other plug-ins, or because the content of the file I opened is too long, or if I use complex regular expressions to match words. I have closed all plug-ins that can be closed, I will give specific details when I encounter this problem next time

This bug may be caused by chromium, because when using chrome, I also encountered a similar problem. This bug usually manifests as: I can normally use the mouse to click, drag and drop, but I cannot use any editing functions, such as input and selection (it is usually possible to select only a small amount of text, but I cannot select a large amount or all of it. Text), but I can perform normal input operations in other applications
